ldes-server-integration-test/src/test/resources/features/fragmentation/fragmentation.feature

```diff
@@ -21,6 +21,27 @@ Feature: LDES Server Fragmentation
       | "data/input/eventstreams/fragmentation/mobility-hindrances.paged.ttl" | "data/input/members/mob-hind.template.ttl"         | "mobility-hindrances" | 617                 | 17        |
       | "data/input/eventstreams/fragmentation/observations/paged.ttl"        | "data/input/members/two-observations.template.ttl" | "observations"        | 1234                | 34        |
 
+  @refragmentation
+  Scenario Outline: Server Can Refragment an LDES
+    Given I create the eventstream <eventStreamDescriptionFile>
+    When I ingest 617 members of template <template> to the collection <collection>
+    Then the LDES <collection> contains <ingestedMemberCount> members
+    And I create the view <viewDescriptionFile>
+    When I fetch the root "paged" fragment of <collection>
+    And I fetch the next fragment through the first "Relation"
+    Then this fragment only has 1 "Relation" relation
+    And this fragment is immutable
+    When I fetch the next fragment through the first "Relation"
+    Then this fragment only has 1 "Relation" relation
+    And this fragment is immutable
+    When I fetch the next fragment through the first "Relation"
+    And this fragment contains <restCount> members
+    And this fragment is mutable
+    And this fragment has no relations
+    Examples:
+      | eventStreamDescriptionFile                                      | viewDescriptionFile                                                       | template                                   | collection            | ingestedMemberCount | restCount |
```

Comment:
No example for state ingestion?

Reply:
honestly it doesn't really matter. I don't want to clog up our ITs with just more spam.. 
This test mainly just checks if pages are made after ingestion took place. state vs version object ingestion doesn't matter at that point anymore